src/commissaire_service/storage/storehandlermanager.py

```diff
@@ -172,17 +188,17 @@ def save(self, model_instance):
         :returns: The saved model instance
         :rtype: commissaire.model.Model
         """
-        logger = self._get_logger()
         handler = self._get_handler(model_instance)
         # Validate before saving
         try:
             model_instance._validate()
         except ValidationError as ve:
-            logger.error(ve.args[0], ve.args[1])
+            self._logger.error(ve.args[0])
+            self._logger.error(ve.args[1])
```

Comment:
We were calling `logger.error()` wrong.  The first argument is supposed to be a format string, subsequent positional arguments are values for that format string.

But for `ValidationError`, `args[0]` is a message about the number of validation errors and `args[1]` is a list of validation error messages.  `logger.error()` choked on it.

I suppose I could format `args[1]` a little nicer, but I was just trying to get rid of a nasty looking traceback that a test case triggered.